src/openklant/components/klantinteracties/api/serializers/partijen.py

```diff
@@ -387,6 +387,7 @@ class Meta:
         validators = []
 
 
+@extend_schema_serializer(deprecate_fields=["andere_partij_identificator"])
```

Comment:
Could you also mark the query parameter as deprecated?

![afbeelding](https://github.com/user-attachments/assets/386cbf76-f657-4562-8145-c72c738f6f23)